package.json

```diff
@@ -22,7 +22,7 @@
     "broccoli-stew": "^2.0.1",
     "ember-cli-babel": "^7.1.2",
     "fs-extra": "^7.0.1",
-    "sharp": "^0.21.1"
+    "sharp": "^0.22.1"
```

Comment:
Looks like Sharp [`0.23.0`](https://github.com/lovell/sharp/releases/tag/v0.23.0) is out now